api/proto/teleport/legacy/types/types.proto

```diff
@@ -4210,6 +4210,12 @@ message LockSpecV2 {
     (gogoproto.nullable) = true,
     (gogoproto.jsontag) = "expires,omitempty"
   ];
+  google.protobuf.Timestamp CreatedOn = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = true,
```

Comment:
`time.Time` is rarely used as a pointer, since it's just a large integer under the hood. Can we remove nullable and allow the zero time to mean unset?

Reply:
Done